File: python/student.py
from degree import Degree
class Student:
    id = None
    First_Name = None
    Last_Name = None
    Email = None
    Age = None
    Student_Degree = Degree
    CourseDays = []

    def __init__(self, id, firstname, lastname, email, age, degree, coursedays):
        self.setID(id)
        self.setFirstName(firstname)
        self.setLastName(lastname)
        self.setEmail(email)
        self.setAge(age)
        self.setDegree(degree)
        self.setCourseDays(coursedays)

    def getID(self): return self.id
    def getFirstName(self): return self.First_Name
    def getLastName(self): return self.Last_Name
    def getEmail(self): return self.Email
    def getAge(self): return self.Age
    def getDegree(self): return self.Student_Degree
    def getCourseDays(self): return self.CourseDays

    def setID(self, id): self.id = id
    def setFirstName(self, name): self.First_Name = name
    def setLastName(self, name): self.Last_Name = name
    def setEmail(self, email): self.Email = email
    def setAge(self, age): self.Age = age
    def setDegree(self, degree): self.Student_Degree = degree
    def setCourseDays(self, days): self.CourseDays = days

    def Print(self):
        print(self.getID()+"   "+self.getFirstName()+"   "+self.getLastName()+"   "+self.getEmail()+"   "+self.getAge()+"   "+str(self.getDegree())+"   "+str(self.getCourseDays()))

File: php/Student.php
<?php
require(__DIR__ ."/Degree.php");
class Student {
    private string $id;
    private string $First_Name;
    private string $Last_Name;
    private string $Email;
    private int $Age;
    private Degree $Student_Degree;
    private array $CourseDays;

    function __construct(string $id, string $firstname,string $lastname,string $email,int $age,Degree $degree,array $coursedays){
        $this->setID($id);
        $this->setFirstName($firstname);
        $this->setLastName($lastname);
        $this->setEmail($email);
        $this->setAge($age);
        $this->setDegree($degree);
        $this->setCourseDays($coursedays);
    }

    function getID() {return $this->id;}
    function getFirstName() {return $this->First_Name;}
    function getLastName() {return $this->Last_Name;}
    function getEmail() {return $this->Email;}
    function getAge() {return $this->Age;}
    function getDegree() {return $this->Student_Degree;}
    function getCourseDays() {return $this->CourseDays;}

    function setID($id) {return $this->id = $id;}
    function setFirstName($name) {return $this->First_Name = $name;}
    function setLastName($name) {return $this->Last_Name = $name;}
    function setEmail($email) {return $this->Email = $email;}
    function setAge($age) {return $this->Age = $age;}
    function setDegree($degree) {return $this->Student_Degree = $degree;}
    function setCourseDays($days) {return $this->CourseDays = $days;}

    function Print() {
        print( $this->getID(). "  ". $this->getFirstName(). "  ".    $this->getLastName(). "  ".    $this->getEmail(). "  ".      $this->getAge(). "  ". $this->getDegree()->name. "  ".      implode( " ", $this->getCourseDays()) ."\n"); 
    }
}
